tests/features/DiscussionTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+use Faker\Factory;
+use Tests\TestCase;
+use LaravelEnso\Core\app\Models\User;
+use Illuminate\Database\Eloquent\Model;
+use LaravelEnso\RoleManager\app\Models\Role;
+use LaravelEnso\Discussions\app\Models\Reply;
+use LaravelEnso\Discussions\app\Enums\Reactions;
+use LaravelEnso\Discussions\app\Models\Reaction;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use LaravelEnso\Discussions\app\Models\Discussion;
+use LaravelEnso\Discussions\app\Traits\Discussable;
+
+class DiscussionsTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private $testModel;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->faker = Factory::create();
+
+        //$this->withoutExceptionHandling();
+
+        $this->seed()
+            ->actingAs(User::first());
+
+        $this->testModel = $this->model();
+    }
+
+    /** @test */
+    public function can_get_discussions_index()
```

Comment:
first create a discussion using the factory, then check that in the index the created discussion is present

Reply:
adressed in cc9077a